maven-model-transform/src/main/java/org/apache/maven/model/transform/stax/BufferingParser.java

```diff
@@ -476,13 +510,20 @@ protected Event bufferEvent() throws XMLStreamException {
                 event.name = pp.getLocalName();
                 event.namespace = pp.getNamespaceURI();
                 event.prefix = pp.getPrefix();
-                // event.text = pp.getText();
                 break;
             case CHARACTERS:
```

Comment:
case fallthrough is risky for code maintenance; better to break explictly for each case